### ISSUE TITLE
Documentation request: Is the index property required?

### ISSUE THREAD

**Issue by DanKaplanSES** [ayonious/console-table-printer#456]
After looking at most of the examples, I don't think an `index` column is *required*, but the majority of the documentation has a column by that name, including [every example on the Adding Rows page](https://console-table.netlify.app/docs/doc-adding-rows). If this column isn't required, I think this is the place to say it. Otherwise, it's implied that it must be manually set by the user.

**Comment by ayonious** [comment]
hi @DanKaplanSES index is not required. You can try out yourself. Here are other examples of this:

https://console-table.netlify.app/docs/doc-computed-function
https://console-table.netlify.app/docs/doc-emojis-special-chars

**Comment by ayonious** [comment]
but I see your point and yes its a better example without using this particular name, which can imply something like this.

**Comment by DanKaplanSES** [comment]
> but I see your point and yes its a better example without using this particular name, which can imply something like this.

Great! Thank you for considering this.

I'd like to add some additional context that may be unnecessary, but might explain how I came to my assumption: I found this library because [`console.table()`](https://developer.mozilla.org/en-US/docs/Web/API/console/table_static) lacked some features I wanted. 

Incidentally, `console.table()` _automatically_ includes an `index` column in its output. Because many `console-table-printer` examples _manually_ include a column with the same name, I thought the `console-table-printer` `index` column may be required as an enhancement to `console.table()`, allowing you to specify an `index` separate from insertion order. 

I want to reiterate I (now) understand that is not the case, but I thought it might help to shed some insight into my thought process. 

Thanks again!
